### ISSUE TITLE
Missing XMPP RFCs?

### ISSUE THREAD

**Issue by Neustradamus** [xsf/rfcs#3]
Can you add all missing XMPP RFCs?

The RFCs list is not complete and maybe not updated for actual (last 2019-06-21):
- https://xmpp.org/rfcs/

Status problems:
- RFC 3920: https://tools.ietf.org/html/rfc3920
- RFC 3921: https://tools.ietf.org/html/rfc3921
- RFC 3922: https://tools.ietf.org/html/rfc3922
- RFC 3923 (no change): https://tools.ietf.org/html/rfc3923
- RFC 4622: https://tools.ietf.org/html/rfc4622
- RFC 4854 (no change): https://tools.ietf.org/html/rfc4854
- RFC 5122: https://tools.ietf.org/html/rfc5122
- RFC 6120: https://tools.ietf.org/html/rfc6120
- RFC 6121 (no change): https://tools.ietf.org/html/rfc6121
- RFC 6122: https://tools.ietf.org/html/rfc6122

New to old not on the website:
- RFC 8600: https://tools.ietf.org/html/rfc8600
- RFC 8284: https://tools.ietf.org/html/rfc8244
- RFC 8266: https://tools.ietf.org/html/rfc8266
- RFC 8048: https://tools.ietf.org/html/rfc8048
- RFC 7712: https://tools.ietf.org/html/rfc7712
- RFC 7702: https://tools.ietf.org/html/rfc7702
- RFC 7700: https://tools.ietf.org/html/rfc7700
- RFC 7622: https://tools.ietf.org/html/rfc7622
- RFC 7590: https://tools.ietf.org/html/rfc7590
- RFC 7573: https://tools.ietf.org/html/rfc7573
- RFC 7572: https://tools.ietf.org/html/rfc7572
- RFC 7395: https://tools.ietf.org/html/rfc7395
- RFC 7340: https://tools.ietf.org/html/rfc7340
- RFC 7259: https://tools.ietf.org/html/rfc7259
- RFC 7248: https://tools.ietf.org/html/rfc7248
- RFC 7247: https://tools.ietf.org/html/rfc7247
- RFC 7165: https://tools.ietf.org/html/rfc7165
- RFC 7081: https://tools.ietf.org/html/rfc7081
- RFC 5437: https://tools.ietf.org/html/rfc5437
- RFC 4979: https://tools.ietf.org/html/rfc4979

Linked to:
- https://github.com/xsf/xmpp.org/issues/414

**Comment by cal0pteryx** [comment]
https://xmpp.org/rfcs/ now features a list of (hopefully) all XMPP related RFCs. This issue can be closed.

**Comment by Neustradamus** [comment]
@cal0pteryx: RFC 4979 and RFC 8048 are missing.

Note: Obsolete or Updated RFCs are always not updated on XMPP.org.

Example:
- https://xmpp.org/rfcs/rfc3920.html
```
NOTHING
```
VS
- https://datatracker.ietf.org/doc/rfc3920/
```
Obsoleted by RFC 6120
Updated by RFC 6122
```

**Comment by cal0pteryx** [comment]
That's wrong. Both https://xmpp.org/rfcs/#4979 and https://xmpp.org/rfcs/#8084 _are_ listed at https://xmpp.org/rfcs/

Note that RFC documents _hosted_ on xmpp.org may not be up to date, and probably never will be. What we do instead: We link to all XMPP-related RFCs hosted on https://datatracker.ietf.org/ which always offers the newest version for each RFC.

This issue can be closed.

**Comment by Neustradamus** [comment]
@cal0pteryx: There is a mistake! ;)
I speak about "RFC 8048", not "RFC 8084" like you.
You must to replace by the good one.

Strange, about RFC 4979, maybe I have not done the good search.

It is needed to update XMPP.org RFC files, people have not informations here about the information about obsolete/update.

**Comment by cal0pteryx** [comment]
RFC 8084 was indeed a twisted number. I've replaced it by RFC 8048 in https://github.com/xsf/xmpp.org/commit/edb1abc8f0fd7b1d11e97627f25cb12704648baa

RFCs hosted at xmpp.org are there for historic purposes. By default, we link to datatracker.ietf.org. As I said, these RFC documents hosted at xmpp.org will probably never be updated.
